README.md

```diff
@@ -12,6 +12,76 @@ This will expose it on port 8080, and store the SQLite db.db in /path/to/storage
 docker build -t tinycalorie .
 docker run -p 8080:80 -v /path/to/storage:/storage tinycalorie
 ```
+## Dockerfile on Dockerhub
+
+### Use the prebuild image on Dockerhub
+[Docker Hub Image](https://hub.docker.com/r/dnshra/tinycalorie)
+```bash
+docker pull dnshra/tinycalorie:1.0
+```
+## Use Docker-Compose File
+1. Install Docker and Docker-Compose
+2. Clone Repository or copy the file below to docker-compose.yml
+3. run docker-compose up -d
+
+[How to install Docker-Compose](https://docs.docker.com/compose/install/)
+
+```
+version: "3.1"
+
+services:
+  tiny-calorie:
+    image: dnshra/tinycalorie:1.0
+    container_name: tiny-calorie
+    volumes: 
+      - /path/to/storage:/app/data
```

Comment:
The db must be stored in `/storage` or it won't be synchronized to the volume.

Neither /path/to/storage or /app/data will work here.

Same goes for the other places you put `/app/data`, I think you got the order wrong.
Isn't the syntax
`/hostpath:/containerpath` ?